client/src/ReviewPictures.css

```diff
@@ -0,0 +1,24 @@
+.pics_container1 {

```

Comment:
I'm not sure if naming conventions would put container or pics first. My first thought it 'pics1_container or container_pics1', but I could be wrong too. Not a big issue.